src/modules/organization/home/OrgHomePage.vue

```diff
@@ -17,13 +17,12 @@ export default defineComponent({
   },
   setup() {
     //Later will read from backend whether user has team
-    const hasTeam = ref(false);
+    const { organizations } = useOrganization();
+    const notHasTeam = computed(
+      () => organizations.value && !organizations.value?.length
+    );

```

Comment:
Why use login notHasTeam? Why not do
!organizations.value && organizations.value?.length !== 0